lib/tasks/serve.js

```diff
@@ -18,6 +32,16 @@ class ServeTask extends Task {
   }
 
   run(options) {
+    let hasBuild = !!options.path;
+
+    if (hasBuild) {
+      if (!fs.existsSync(options.path)) {
+        throw new SilentError(`The path ${options.path} does not exist. Please specify a valid build directory to test.`);
```

Comment:
`Please specify a valid build directory to test.` should likely be `Please specify a valid build directory to serve.`